src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -538,6 +538,24 @@ public void testShowStats() {
 
     }
 
+    @Test
+    public void testIndicateMissingResponses() {
+
+        ______TS("Typical case: Hide Missing Responses");
+
+        assertEquals(resultsPage.indicateMissingResponsesCheckbox.getAttribute("checked"), "true");
+        assertFalse(resultsPage.verifyMissingResponsesVisibility());
+
+        resultsPage.clickIndicateMissingResponses();
+        assertEquals(resultsPage.indicateMissingResponsesCheckbox.getAttribute("checked"), null);
+        assertTrue(resultsPage.verifyMissingResponsesVisibility());
+
+        resultsPage.clickIndicateMissingResponses();
+        assertEquals(resultsPage.indicateMissingResponsesCheckbox.getAttribute("checked"), "true");
```

Comment:
Might be possible to replace these with `isSelected()` eg.

`assertTrue(resultsPage.indicateMissingResponsesCheckbox.isSelected());`

although I doubt this is related to the current issue with testing.